### PR TITLE
Update documentation on enabling extension manager

### PR DESCRIPTION
**Pull request by bobmayuze**
Updated documentation on enabling extension manager.

### CODE REVIEW

File: docs/source/user/extensions.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ Using the Extension Manager
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To manage your extensions, you can use the extension manager. By default, the
-manager is disabled, but you can enable it with the following steps:
+manager is disabled. You can enable it by searching **Extension Manager** in the command platte. You can also enable it with the following steps:
```
Comment (jtpio) [inline docs/source/user/extensions.rst]:
```suggestion
manager is disabled. You can enable it by searching **Extension Manager** in the command palette. You can also enable it with the following steps:
```

**Comment by jtpio** [comment]
Thanks!

Do you think it would make sense to also add a screenshot?

**Comment by bobmayuze** [comment]
> Thanks!
> 
> Do you think it would make sense to also add a screenshot?

Yeah, definitely.

File: docs/source/user/extensions.rst [hunk 2]
```diff
@@ -49,7 +49,15 @@ Using the Extension Manager
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To manage your extensions, you can use the extension manager. By default, the
-manager is disabled, but you can enable it with the following steps:
+manager is disabled. You can enable it by searching **Extension Manager** in the command platte.
```
Comment (jtpio) [inline docs/source/user/extensions.rst]:
```suggestion
manager is disabled. You can enable it by searching **Extension Manager** in the command palette. 
```

File: docs/source/user/extensions.rst [hunk 3]
```diff
@@ -49,7 +49,15 @@ Using the Extension Manager
...
 
 To manage your extensions, you can use the extension manager. By default, the
-manager is disabled, but you can enable it with the following steps:
+manager is disabled. You can enable it by searching **Extension Manager** in the command platte. 
+
+.. figure:: images/extension_manager_enable_manger.png
+   :align: center
+   :class: jp-screenshot
+
+   **Figure:** Enable extension manager by searching in the command platte
```
Comment (jtpio) [inline docs/source/user/extensions.rst]:
```suggestion
   **Figure:** Enable extension manager by searching in the command palette
```

File: docs/source/user/extensions.rst [hunk 4]
```diff
@@ -49,7 +49,15 @@ Using the Extension Manager
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To manage your extensions, you can use the extension manager. By default, the
-manager is disabled, but you can enable it with the following steps:
+manager is disabled. You can enable it by searching **Extension Manager** in the command platte. 
+
+.. figure:: images/extension_manager_enable_manger.png
```
Comment (jtpio) [inline docs/source/user/extensions.rst]: Quick typo in the name of the file (file should also be renamed):
```suggestion
.. figure:: images/extension_manager_enable_manager.png
```

Comment (bobmayuze) [reply docs/source/user/extensions.rst]: Sorry for the typo. I'll get it fixed rn.

**Comment by jasongrout** [comment]
Thanks! Did you compress your screenshot with pngquant? (see screenshot instructions at https://jupyterlab.readthedocs.io/en/stable/developer/documentation.html#screenshots-and-animations)

**Comment by blink1073** [comment]
I compress the screen shot.  Thanks again, @bobmayuze!

**Pull request merged**